deployments-ant-task/pom.xml

```diff
@@ -1,14 +1,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <parent>
     <artifactId>deployments-parent</artifactId>
     <groupId>org.ccci</groupId>
-    <version>3-SNAPSHOT</version>
+    <version>3</version>
   </parent>
-  <groupId>org.ccci</groupId>
```

Comment:
If you're cleaning up, you could also remove https://github.com/CruGlobal/deployments/blob/master/deployments-maven-plugin/pom.xml#L5 and https://github.com/CruGlobal/deployments/blob/master/deployments-command-line/pom.xml#L3.

Reply:
I did, actually. Those links point to master, not to my branch.

Reply:
Ahh, ok. I did know I sent you links to master, but I thought for some reason I'd seen them in your branch. My mistake. I'll just send links to the branch in future.